### ISSUE TITLE
[indent] error indent at template types

### ISSUE THREAD

**Issue by lext-7** [typescript-eslint/typescript-eslint#658]
**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/indent": "error"
  }
}
```

```TS
readonly requestFn: IApiPagingCrawlMixinOptions<
    TUserData,
    TResponse
>['requestFn'];
```

**Expected Result**

It will raise indent error:

```TS
readonly requestFn: IApiPagingCrawlMixinOptions<
    TUserData,
    TResponse
>['requestFn'];
```

**Actual Result**

```TS
readonly requestFn: IApiPagingCrawlMixinOptions<
TUserData,
TResponse
>['requestFn'];
```

**Additional Info**

**Versions**

| package                            | version |
| ---------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin` | `1.11.0"` |
| `@typescript-eslint/parser`        | `1.11.0"` |
| `TypeScript`                       | `3.5.2` |
| `ESLint`                           | `6.0.1` |
| `node`                             | `12.2.0` |
| `npm`                              | `6.9.0` |

**Comment by lext-7** [comment]
When will it be fixed ? Is there a promised time ?

**Comment by bradzacher** [comment]
If you need it sooner rather than later, then feel free to raise a PR - we encourage and appreciate all contributions.

Otherwise a rewrite of the indent rule is on the cards, but it's a very, very complex rule and will take a long while.
I don't have the time to dedicate to figuring out the eslint base code and extending it with typescript nodes, so right now it's on the backlog.

**Comment by jacob87o2** [comment]
I think this is the same issue: #455

**Comment by bradzacher** [comment]
it is actually, thanks for pointing that out.
